workspaces/diff-engine/cli/src/main.rs

```diff
@@ -71,16 +71,13 @@ fn main() {
 
   let spec_projection = Arc::new(SpecProjection::from(events));
 
-  let mut runtime_builder = tokio::runtime::Builder::new();
+  let mut runtime_builder = tokio::runtime::Builder::new_multi_thread();
   runtime_builder.enable_all();
-  runtime_builder.threaded_scheduler();
   if let Some(core_threads) = core_threads_count {
-    runtime_builder
-      .max_threads((core_threads as usize) * 2)
-      .core_threads(core_threads as usize);
+    runtime_builder.worker_threads(core_threads as usize);
```

Comment:
They got rid of the `max_thread` and `core_threads` because they were confusing. That they were!